### ISSUE TITLE
Diffrent beetween pythin-telegram-bot and native telegram api

### ISSUE THREAD

**Issue by xmen1412** [python-telegram-bot/python-telegram-bot#1969]
Hi There.

I Have a couple and i was wondering if you guys can answer my Question :D 👍

1. can i only use this library extension for creating simple bot for only send the data from my local server to the client who subscribe my bot without Pure Telegram API And Flask Framework (fyi many tutorial that i found usually use Flask and Url request as well as Webhook method to communicate with telegram )? 

2. is this competible with pyserial? because, i have an weather station arduino project with serial communication that use telegram app to pass information about Sensor to Telegram Chat Client

3. what the diffrence this extension and pure native telegram api

**Comment by Bibo-Joshi** [comment]
Let me try to answer the question in reversed order:

3. The "pure API" provides only the methods/classes defined by the Telegram API. `telegram.ext` provides an easy to use interface. E.g. you don't have to worry about how to get your updates. See [here](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Extensions-–-Your-first-Bot) and also compare `echobot.py` and `echobot2.py` in the examples section.

2. What do you mean by "compatible"? This lib is just another python module you can install with pip. It should be no problem to import both ptb and pyserial in a script and send messages depending on what the serial thingy does

1. About Flask & requests: see 1. The rest of the question I don't really understand. Can you rephrase? Please note that Telegram Bots are not meant to send anything to servers, but to Telegram users via the Telegram instant messenger.
